src/main/java/ai/promoted/delivery/client/PropertiesFactory.java

```diff
@@ -0,0 +1,73 @@
+package ai.promoted.delivery.client;
+
+import java.util.Map;
+import com.google.protobuf.ListValue;
+import com.google.protobuf.NullValue;
+
+import ai.promoted.proto.common.Properties;
+
+/**
+ * Helper methods to create Properties.
+ */
+public class PropertiesFactory {
+  /**
+   * Create a properties builder with a struct equivalent to the given map.
+   * @param map the properties as a map
+   * @return a populated Insertion
```

Comment:
Snit - please update the javadoc @return comments from Insertion to Properties.  Ditto down below.

Reply:
Done